_posts/2018-03-29-new-profiles.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: New Profiles
+date: 2018-03-29
+categories:
+  - projects
+description: New CSH profiles in Flask because the old one broke
+image: https://i.imgur.com/KGNp3xX.png
+image-sm: https://i.imgur.com/KGNp3xX.png
+author: Nate Dooley
+author-image: https://profiles.csh.rit.edu/image/dooley
+author-bio: Freshman CIT major
+author-email: dooley@csh.rit.edu
+author-social:
+  github: https://github.com/natedooley
+---
+
+CSH Profiles is the Premier Social Network for Computer Science House members. It facilitates social interaction the way CSHers like it: not interacting at all and just watching from a distance! Using Profiles, you can find all your favorite members and look at all their information, as well as their picture if you forgot what they looked like. You can find their birthday and remember to tell them "Happy Birthday!" or, you can find their phone number and sign it up for numerous spam agencies as a HILARIOUS prank! Profiles has it all, so [sign up](https://profiles.csh.rit.edu) today!
+
+Prior to writing Profiles, I had never written a single line of Python in my life. I found that I really like using Flask and making web sites do cool things with a real backend framework. I've spent easily 20+ hours just learning things about the language and CSH services. I learned a little bit of Javascript and AJAX in the search and editing functions, as well as using raw LDAP to upload images. The project is in the main CSH github repo for Profiles under the v2 branch, and will replace master once it's complete.   
```

Comment:
`s/main CSH github repo for Profiles/[CSH profiles repository](https://github.com/ComputerScienceHouse/profiles/tree/v2)/`

Reply:
Also, is this still the case? There was some discussion about it on Slack and if we can resolve it before publishing this post you should update the post to reflect that.